src/sentry/web/debug_urls.py

```diff
@@ -65,67 +65,67 @@
 from sentry.web.frontend.debug.debug_unassigned_email import DebugUnassignedEmailView
 
 urlpatterns = [
-    url(r"^debug/mail/alert/$", sentry.web.frontend.debug.mail.alert),
-    url(r"^debug/mail/note/$", DebugNoteEmailView.as_view()),
-    url(r"^debug/mail/new-release/$", DebugNewReleaseEmailView.as_view()),
-    url(r"^debug/mail/new-user-feedback/$", DebugNewUserFeedbackEmailView.as_view()),
-    url(r"^debug/mail/assigned/$", DebugAssignedEmailView.as_view()),
-    url(r"^debug/mail/assigned/self/$", DebugSelfAssignedEmailView.as_view()),
-    url(r"^debug/mail/assigned/team/$", DebugSelfAssignedTeamEmailView.as_view()),
-    url(
-        r"^debug/mail/codeowners_auto_sync_failure/$", DebugCodeOwnersAutoSyncFailureView.as_view()
+    re_path("^debug/mail/alert", sentry.web.frontend.debug.mail.alert),
+    re_path("^debug/mail/note", DebugNoteEmailView.as_view()),
+    re_path("^debug/mail/new-release", DebugNewReleaseEmailView.as_view()),
+    re_path("^debug/mail/new-user-feedback", DebugNewUserFeedbackEmailView.as_view()),
+    re_path("^debug/mail/assigned/self", DebugSelfAssignedEmailView.as_view()),
+    re_path("^debug/mail/assigned/team", DebugSelfAssignedTeamEmailView.as_view()),
+    re_path("^debug/mail/assigned", DebugAssignedEmailView.as_view()),
```

Comment:
Is this re-ordering necessary to make sure this doesn't capture the `self` and `team` paths?